apps/docs/guide/deploying/self-hosted.md

```diff
@@ -0,0 +1,164 @@
+
+-   [Alchemy](https://www.alchemy.com)
+-   [Infura](https://infura.io)
+-   [Ankr](https://www.ankr.com)
+-   [QuickNode](https://www.quicknode.com)
+-   [GetBlock](https://getblock.io)
+
+Each service has its own characteristics and pricing, and the developer should choose the one that best fits his needs. Any of them should work with the rollups node, but the first two are the most battle-tested.
+
+The URLs for the HTTP and WebSocket endpoints must be configured as environment variables called `CARTESI_BLOCKCHAIN_HTTP_ENDPOINT` and `CARTESI_BLOCKCHAIN_WS_ENDPOINT`, respectively.
+
+::: warning
+The Web3 provider URLs can be sensitive information, if they are private URLs the developer is paying for. The developer should take care to keep it secure, like using a secret mechanism provided by the cloud provider of choice.
+:::
+
+## Cloud server
+
+The developer will need a server to host the application node and run it 24/7. This server will expose a single port to the internet, so client applications, like front-ends, can consume the rollups node APIs through [GraphQL](https://docs.cartesi.io/cartesi-rollups/api/graphql/index/) or [Inspect](https://docs.cartesi.io/cartesi-rollups/api/inspect/inspect/) requests.
+
+The server minimum requirements will depend on the expected usage of the application and on the specifications of the Cartesi machine in use, like its RAM size and total size. The developer will have to experiment with different configurations to find the best fit for his needs. We suggest starting with a minimum of 8GB of RAM, and scaling up vertically as needed.
+
+The Cartesi rollups node is distributed as a Docker image, so the server will need to have Docker installed. Any popular cloud provider, like AWS, GCP, Azure, Digital Ocean, or Linode, is capable of running docker containers, either manually by spawing a server and running docker, or by using a managed container infrastructure like Kubernetes.
+
+The developer can also use a service like [Fly.io](https://fly.io) to deploy the application node.
+
+We will describe below two methods of running a rollups node, the first using Docker, which can be used as the basic information to use any cloud provider, and the second using [Fly.io](https://fly.io), a managed service for running Docker containers.
+
+### Using Docker in any cloud provider
+
+Assuming the developer completed the above requirements, and has a complete `.env` file, he can run the rollups node using the following command:
+
+```shell
+$ docker run --env-file <env-file> -p 10000:10000 <image-id>
+(...)
+INFO http: listening at [::]:10000
+(...)
+INFO rollups-node: all services are ready
+(...)
+```
+
+In the examples above the `<env-file>` file is `0xc87999b8a93609268b10de25f2e49d35f80fad92813310edc585ed644a9805d3.env` and the `<image-id>` is `sha256:5c355a9bddc92aa08987f395a257a0b32a51552c969eb161386e46f9380ea2ac`.
```

Comment:
Maybe a disclaimer the developer need top push the image to a container registry accessible by the cloud provider where the command will be executed.